sanic/app.py

```diff
@@ -840,7 +846,13 @@ async def handle_request(self, request: Request):  # no cov
 
         # Define `response` var here to remove warnings about
         # allocation before assignment below.
-        response = None
+        response: Optional[
+            Union[
+                BaseHTTPResponse,
+                Coroutine[Any, Any, Optional[BaseHTTPResponse]],
+            ]
+        ] = None
+        run_middleware = True
```

Comment:
Looks like this variable is being set to False twice (one here and one in line 900)

Reply:
Once as `True`, once as `False`. Correct?